src/style.css

```diff
@@ -0,0 +1,27 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  width: 96px;
```

Comment:
why you need to set `width: 96px` ? you have the size for each star (`16px`) so let them stretch the parent container